SR2019/2019-01-26-competition-date-announcement.md

```diff
@@ -0,0 +1,26 @@
+---
+to: Student Robotics 2019
+subject: Confirming the dates for Student Robotics 2019
+---
+
+Hello all,
+
+As some eagle-eyed members of your team may have noticed on [the forum](https://studentrobotics.org/forum/viewtopic.php?f=9&t=35), today's the day we confirm the dates for Student Robotics 2019!
+
+The competition will be held from *Saturday 6th April* until *Sunday 7th April*. The competition will be held at the University of Southampton's Highfield campus, so please plan accommodation / transport accordingly. The exact start/end times are to be confirmed, but doors open 9am, and we aim to finish around 5pm.
+
+[Competition details](https://studentrobotics.org/events/sr2019/competition/)
+
+## Tech Days
+
+Tech Days are opportunities for you to get help if you're struggling with a technical issue, and is a controlled environment to make more progress on your robots. It’s also an opportunity to see how well you're doing in comparison to other teams. The remaining tech days are:
+
+- 9th February
+- 9th March
+
+Information for these tech days are available on [our website](https://studentrobotics.org/events/).
+
+## Media Consent
```

Comment:
nit: just checking that whatever renders this is ok with the lack of empty line after this heading?

Reply:
It's in markdown for review convenience. I manually copy and re-format the text so it looks as expected in MailChimp.